### PR TITLE
fix: handle untranslated custom enduser consent scopes

### PR DESCRIPTION
**Pull request by sherizada-okta**
## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [x] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-389469](https://oktainc.atlassian.net/browse/OKTA-389469)

### CODE REVIEW

File: src/v2/view-builder/views/consent/AdminConsentView.js [hunk 1]
```diff
@@ -2,23 +2,34 @@ import { BaseView } from '../../internals';
...
-      displayName: (isAdminConsent || name === 'openid') ? name : loc(`${i18nKeyPrefix}.${name}.label`, 'login'),
-      description: loc(`${i18nKeyPrefix}.${name}.desc`, 'login'),
-    }));
+    const localizedScopes = scopes.map(({ name, displayName, description }) => {
+      const scopeKey = `${i18nKeyPrefix}.${name}`;
+      const labelKey = `${scopeKey}.label`;
+      const descKey = `${scopeKey}.desc`;
+      const doesLabelExist = doesI18NKeyExist(labelKey);
+      const doesDescExist = doesI18NKeyExist(descKey);
+      const i18nDisplayName = doesLabelExist ? loc(labelKey, 'login') : displayName;
```
Comment (nikhilvenkatraman-okta) [inline src/v2/view-builder/views/consent/AdminConsentView.js:23]: if `doesLabelExist` is false, will `displayName` will get rendered in English ? . Same for `doesDescExist` ?

Comment (nikhilvenkatraman-okta) [reply src/v2/view-builder/views/consent/AdminConsentView.js:23]: And  if I understand correctly, for those cases `isCustomized` is true which adds the `no-translate` class ?

Comment (sherizada-okta) [reply src/v2/view-builder/views/consent/AdminConsentView.js:23]: Yes, so if the `displayName` has been customized (as can happen on enduser, not admin) I'll add the no-translate class.

**Review by nikhilvenkatraman-okta — APPROVED** [review]
Looks good, can we add any tests to verify this change ?

**Pull request closed without merging**